src/pages/MainPage.css

```diff
@@ -0,0 +1,37 @@
+header {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.5rem;
+  width: 95vw;
+  min-width: 280px;
+  color: white;
+}
+
+.Header__topImage {
+  height: 65px;
+
+  background: linear-gradient(300.63deg, #412027 45.56%, #562736 85.31%);
+  border: 2px solid #000000;
+  box-sizing: border-box;
+  border-radius: 30px;
+}
+
+.Header__NavBar {
```

Comment:
Header__NavBar but previously we have `Header__topImage`

Be consistent  with your naming. 
Also `NavBar` could be its own block!, you don't need to have it as part of the `Header`

Reply:
Thanks for pointing out that mistake!
And ok, I will think about having the navigation bar not part of the header.